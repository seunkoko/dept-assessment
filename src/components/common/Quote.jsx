// eslint-disable-next-line
import React from 'react';

import './common.scss';

const Quote = (props) => {
	return (
    <>
      <div className="container-fluid quote-style">
        <p>
          “Dept helped us tell our story through a bold new identity 
          and a robust online experience. To the tune of 60% growth 
          in online bookings in just one month.”
        </p>
        <p>
          MATTIJS TEN BRINK – CEO, TRANSAVIA
        </p>
      </div>
    </>
	);
};

export default Quote;
