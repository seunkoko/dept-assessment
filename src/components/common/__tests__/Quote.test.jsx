import React from 'react';

import { shallow } from 'enzyme';

import Quote from '../Quote';

describe('Quote', () => {
	const component = shallow(<Quote />);

	it('should render Quote component', () => {
		expect(component).toMatchSnapshot();
  });
  
});