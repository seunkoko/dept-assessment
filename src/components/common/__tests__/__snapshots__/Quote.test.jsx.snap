// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render Quote component 1`] = `
<Fragment>
  <div
    className="container-fluid quote-style"
  >
    <p>
      “Dept helped us tell our story through a bold new identity and a robust online experience. To the tune of 60% growth in online bookings in just one month.”
    </p>
    <p>
      MATTIJS TEN BRINK – CEO, TRANSAVIA
    </p>
  </div>
</Fragment>
`;
